evennia/accounts/accounts.py

```diff
@@ -189,6 +191,41 @@ def nicks(self):
     @lazy_property
     def sessions(self):
         return AccountSessionHandler(self)
+        
+    def get_absolute_url(self):
+        """
+        Returns the canonical URL for an Account. 
+        
```

Comment:
You need a `Returns:` docstring heading here. Would be good with an `Example:` block too, giving an example of how the url actually will look like. Same for the other methods below.